Sources/ModelValidator/Validators/StartsWithValidator.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+
+extension Validator where T: Collection, T.Element: Equatable {
+    /// Checks if the validated collection starts with the given sequence.
+    ///
+    /// - Parameter sequence: A sequence that have to be at the beginning of the validated collection.
```

Comment:
Parameter sequence: A sequence that **has** to be at the beginning of the validated collection.